.github/workflows/build-python.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         name:
           [
             ubuntu-20.04-gcc-9,
-            ubuntu-22.04-gcc-9-tbb,
```

Comment:
Why this change?

Reply:
This change was made in a previous PR to fix the CI hanging on Python+TBB, however it doesn't seem to have made a difference as the issue still persists.

A lot of people still use Ubuntu 20.04 since it is the last version that supports ROS1 so having a CI environment for it makes more sense (plus backwards compatibility).